README.md

```diff
@@ -112,6 +114,8 @@ To install this library via [npm](https://www.npmjs.org) **version 3**, use the
 npm install @reactivex/rxjs
 ```
 
+This will include all builds for CJS/ES6/AMD/TypeScript.
```

Comment:
For now it's only `cjs/global`. I don't think instruction to use `rjxs` need to be removed though.